lib/commands/web.js

```diff
@@ -302,6 +311,7 @@ extensions.waitForAtom = async function waitForAtom (promise) {
       // check if there is an alert
       if (await this.checkForAlert()) {
```

Comment:
I'm not quite sure how this is going to work together with auto alert accept setting